src/main/resources/application-postgres.properties

```diff
@@ -1,4 +1,4 @@
-spring.datasource.url=jdbc:postgresql://${DATABASE_ENDPOINT}/${DATABASE_NAME}
+spring.datasource.url=jdbc:postgresql://${DATABASE_ENDPOINT}/${DATABASE_NAME}?sslmode=verify-full
```

Comment:
this looks naff - I would have expected a `DATASOURCE_URL` environment variable then these sort of production settings don't need to be here